### PR TITLE
Improve fork database logging

### PR DESCRIPTION
**Pull request by heifner**
- Change a `warn` level log to `info` since it does not necessarily indicate any issue.
- Add block ids to fork database log output.
- Add additional fork database log output at shutdown.

### CODE REVIEW

File: libraries/chain/fork_database.cpp [hunk 1]
```diff
@@ -719,8 +722,7 @@ namespace eosio::chain {
       auto in_use_value = in_use.load();
       // check that fork_dbs are in a consistent state
       if (!legacy_valid && !savanna_valid) {
-         wlog( "fork_database is in a bad state when closing; not writing out '${filename}', legacy_valid=${l}, savanna_valid=${s}",
-               ("filename", fork_db_file)("l", legacy_valid)("s", savanna_valid) );
+         ilog("No fork_database to persist, not writing out: ${f}", ("f", fork_db_file));
```
Comment (linh2931) [inline libraries/chain/fork_database.cpp:725]: Since we are not writing out, we don't need to log the file name `fork_db_file`.

Comment (heifner) [reply libraries/chain/fork_database.cpp:725]: I thought it didn't hurt to include the file. If you think it is confusing, I can remove it.

Comment (linh2931) [reply libraries/chain/fork_database.cpp:725]: I think it is a little confusing. The user might think an empty file is there. But  this is very minor, you don't need to change.

**Review by linh2931 — APPROVED** [review]

File: libraries/chain/fork_database.cpp (hunk 1 above)
Comment (heifner) [reply libraries/chain/fork_database.cpp:725]: Makes sense, I'll remove it

**Review by spoonincode — APPROVED** [review]

**Comment by ericpassmore** [comment]
Note:start
group: STABILITY
category: LOGGING
summary: Improve fork database logging, additional data added at shutdown.
Note:end

**Pull request merged**